src/styles/mixins/colorMixins.ts

```diff
@@ -12,4 +12,16 @@ const bodyTextColor = (theme: DefaultTheme) => {
    `
 }
 
-export { bodyBackgroundColor, bodyTextColor }
+const headerBackground = (theme: DefaultTheme) => {
```

Comment:
For these mixins shouldnt we just pass the color directly instead of passing the entire theme object. Lets say there are some elements in the UI which dont need to be controlled from theme.

For example if we have to show red bsckground somewhere and is independent of theme then would we create new mixin for that?

Reply:
Usually as a standard so that every mixin has same parameter, usually entire theme object is passed and maybe in future if mixin needs other parameters from theme object, there is no need to add parameters.

Reply:
And if there is theme, then all colors, fonts etc all should come from theme. No hard coded values should be there